pkg/detectors/infobip/infobip.go

```diff
@@ -0,0 +1,82 @@
+package infobip
```

Comment:
Please add the tags back. You can run the test by `go test ./pkg/detectors/infobip -tags=detectors`
```
//go:build detectors
// +build detectors
```